Composer/packages/client/src/utils/pageLinks.ts

```diff
@@ -57,7 +57,7 @@ export const topLinks = (
       disabled: !botLoaded,
     },
     {
-      to: `/bot/${projectId}/diagnostics`,
+      to: `/bot/${rootProjectId || projectId}/diagnostics`,
```

Comment:
shouldn't this be prioritizing projectId over the rootProjectId? meaning if projectId is there us it, otherwise fall back on the rootProjectId (```projectId || rootProjectId```)

Reply:
Hi, @hatpick The projectId here I think is skillId or rootProjectId, so if the rootProjectId is null, the projectId is root id.